utils/FitUtils.py

```diff
@@ -245,6 +248,22 @@ def _Pol3IntegralNorm(self, x, par):
         fourthTerm = par[3] * (x[0]**3 - 1 / 4. * (self.maxMass**4 - self.minMass**4) / (self.maxMass - self.minMass))
         return firstTerm + secondTerm + thirdTerm + fourthTerm
 
+    def _ExpoPowIntegralNorm(self, x, par):
```

Comment:
Please specify that this is intended to work only for the D*

Reply:
Done